### DISCUSSION TITLE
zwavejs2mqtt (Docker) migration guide

### DISCUSSION THREAD

**Discussion by begincalendar** [zwave-js/zwave-js-ui#2742] [Q&A] [answered]
Hi there,

I'm currently using the [zwavejs/zwavejs2mqtt:6.9.0](https://hub.docker.com/layers/zwavejs/zwavejs2mqtt/6.9.0/images/sha256-07722c83a274813e9934e147a71865eee369c0dfbc09a9801e6a2312acf28032?context=explore) Docker image and discovered that the (zwavejs2mqtt) Docker image is now deprecated.

I have noticed [this migration page](https://zwave-js.github.io/zwave-js-ui/#/guide/migrating), but that reads more of a blur between "breaking changes" vs. "migration guide".

Is there a more comprehensive (or beginner-friendly) guide (e.g. one for those who have only interacted with the GUI so far) on how to migrate from zwavejs2mqtt to zwave-js-ui?

**Comment by kpine** [comment, marked as answer]
That migration page is for migrating from **zwave2mqtt**, which is the old OZW-based application. See [v8.0.0 release notes](https://github.com/zwave-js/zwave-js-ui/releases/tag/v8.0.0) for instructions. Unless you have a highly customized installation, you will only need to change the image name.

**Reply by begincalendar** [reply]
Oh indeed (I misread the page). Thank you!
